packages/build/tests/core/config/tests.js

```diff
@@ -1,6 +1,7 @@
 const test = require('ava')
 
-const { runFixture, runFixtureConfig, FIXTURES_DIR, getJsonOpt, escapeExecaOpt } = require('../../helpers/main')
+const { runFixture, FIXTURES_DIR } = require('../../helpers/main')
+const { runFixture: runFixtureConfig, getJsonOpt, escapeExecaOpt } = require('../../../../config/tests/helpers/main')
```

Comment:
We could probably do something like this:
https://github.com/netlify/netlify-cms/blob/a91e8316f7d4618f2fa2a3d94bc733b5a5d3ade8/jest.config.js#L3
To help with referencing cross packages code in tests.

Not really necessary for this PR

Reply:
Yes definitely should do this :+1: 

We could also abstract the test utility into its own package.